### ISSUE TITLE
Transfer breakout room whiteboard back to the main session

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#9272]
**Is your feature request related to a problem? Please describe.**
When users are put into a breakout room and asked to collaborate on the whiteboard, we want the contents of the whiteboard to be brought back to the main room

**Describe the solution you'd like**
When the breakout rooms end, there should be a slide deck created for each breakout room. These slide decks appear in the main room and are accessible by the presenter.

**Comment by ostasevych** [comment]
As a workaround I am asking to add opportunity to copy and paste images and attach files in Shared notes and Common chat. Where you may place printscreenned images from whiteboards.

**Comment by tbussmann** [comment]
@stale bot: please keep this request.

Another workaround for this could be reached by #4882: allow the users to download their breakout whiteboards and upload / share them again in the group

**Comment by ffdixon** [comment]
Yes, once we have #4882 in place,  we can implement annotations from breakout rooms coming back to the main room (so students can present), and annotations from the current slide going to the breakout room.  These won't all be implemented at once, but #4882 is the required feature to implement them.
